client/app/sessions/sessions.js

```diff
@@ -76,7 +74,7 @@ angular.module('dinnerDaddy.sessions', [])
 
 
   $scope.getFriends = function (user) {
-    console.log($cookies)
+    console.log($cookies);
```

Comment:
you don't need to do this change--I'm writing this part right now, so don't want to have a change here that's going to cause merge conflicts